ci-operator/step-registry/cucushift/upgrade/check-not-recommended/cucushift-upgrade-check-not-recommended-commands.sh

```diff
@@ -20,6 +20,7 @@ function extract_oc(){
     mv ${tmp_oc}/oc ${OC_DIR} -f
     which oc
     oc version --client
+    export PATH="$PATH"
```

Comment:
should it be moved to ahead of `oc version --client` so we will know if it takes effect?

Reply:
according to my debugging, it should. good catch!

Reply:
ah, good catch, pasted it to a wrong line.

Reply:
Updated, the new update should be low risky, so I will not re-run pj-rehearse jobs.